core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.ts

```diff
@@ -97,12 +97,10 @@ export class LearnerDashboardIconsComponent implements OnInit, OnChanges {
       } else {
         if (this.isContainerNarrow) {
           return true;
-        } else {
-          return this.activityIsCurrentlyHoveredOver;
         }
       }
     }
-    return false;
+    return true;
```

Comment:
I guess you could replace all this with a single `return true;` in the (first) else block (since it now returns true regardless of whether the container is narrow or not)?

Reply:
Done!